cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -461,11 +461,10 @@ func (p *dockerProject) packBuild(
 
 		if svc.Language == ServiceLanguageJava {
 			environ = append(environ, "ORYX_RUNTIME_PORT=8080")
-			// Consider it as multi-module project if service path is not same as its project path
 			// For multi-module project, specify parent directory and submodule for pack build
-			if svc.Path() != svc.Project.Path {
-				buildContext = svc.Project.Path
-				svcRelPath, err := filepath.Rel(svc.Project.Path, svc.Path())
+			if svc.ParentPath != "" {
```

Comment:
Seems like this property only applies to Java projects, I am wondering whether it's appropriate to expose this option to all language users.

Reply:
Currently it only exposes to java, because we have some code to detect parent path. I'm not sure if other languages cover the parent path detection like java, if it's ready, I think it can be exposed to all languages.

Reply:
Moved this option to all languages, as it doesn't block existing logic (`parentPath` is empty by default). If other languages want to use this, they should prepare the `parentPath` property properly.